postfix-3.9.yaml

```diff
@@ -0,0 +1,216 @@
+package:
+  name: postfix-3.9
+  version: 3.9.0
+  epoch: 0
+  description: Secure and fast drop-in replacement for Sendmail (MTA)
+  copyright:
+    - license: IPL-1.0 OR EPL-2.0
+  dependencies:
+    provides:
+      - postfix=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - coreutils
+      - cyrus-sasl-dev
+      - icu-dev
+      - libnsl-dev
+      - linux-headers
+      - lmdb-dev
+      - m4
+      - mariadb-connector-c-dev
+      - openldap-dev
+      - openssl-dev
+      - pcre2-dev
+      - perl
+      - postgresql-dev
+      - sqlite-dev
+
+data:
+  - name: _subpackages
+    items:
+      ldap:
+      mysql:
+      pcre:
+      pgsql:
+      sqlite:
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 56f5e420e7c25455a4e96c19b672f80f9a0a35fb5becc9247c9e3d5dcc617f34
+      uri: https://de.postfix.org/ftpmirror/official/postfix-${{package.version}}.tar.gz
+
+  - runs: |
+      _mvline() {
+        # Assigning arguments to variables
+        local regex="$1"
+        local outfile="$3.d/$2"
+        local infile="$3"
+
+        # Use sed to find the line matching the regex, write it to outfile, and delete it from infile
+        sed -i -E -e "\|$regex|{
+      w $outfile
+      d
+      }" "$infile"
+      }
+
+      addgroup -S postfix 2>/dev/null
```

Comment:
Will the image need this user+group too?

Reply:
this is package only type of request afaik